cypress/integration/deal-groups.js

```diff
@@ -0,0 +1,257 @@
+        .wait('@atd-api')
+        .wait('@role')
+        .wait('@sessionUser'); // making sure these APIs are cleared
+
+
+      // Buyers Section
+      cy.get('[data-qa="deal-group-form--select-bidder"]').click().focus().type('A');
+      cy.get('[data-qa="deal-group-form--select-bidder"]').parent('.input-wrapper--typeahead').find('.dropdown-menu').children('li')
+        .first()
+        .children('a')
+        .click();
+      cy.get('[data-qa="deal-form--select-advertiser"]').click().type('max effort');
+      cy.contains('max effort').click(); // clicking on advertiser dropdown
+
+      // // Team Member Section
+      cy.get('[data-qa="deal-group-form--select--agency_sales_lead"]').click();
+      cy.get('ul > li.active', { timeout: 9000 }).click(); // clicking on 1st option from dropdown
+      cy.get('[data-qa="deal-group-form--select--client_service_manager"]').click();
+      cy.get('ul > li.active', { timeout: 9000 }).click(); // clicking on 1st option from dropdown
+      cy.get('[data-qa="deal-group-form--select--campaign_manager"]').click();
+      cy.get('ul > li.active', { timeout: 9000 }).click(); // clicking on 1st option from dropdown
+
+      // KPI Section
+      cy.get('[data-qa="deal-group-form--optimizable-kpi"]').focus().click();
+      cy.get('[data-qa="deal-group-form--optimizable-kpi--0"]').click();
+      cy.get('[data-qa="deal-group-form--outcome-kpi"]').click();
+      cy.get('[data-qa="deal-group-form--outcome-kpi--0"]').click();
+
+      // Submitting Deal Group info
+      cy.get('[data-qa="deal-group-add-edit--submit"]').click();
+      cy.get('[data-qa="modal--confirm"]').click();
+      cy.get('[data-qa="deal-group-detail--title"]', { timeout: 5000 }).should('contain', dealGroupName);
+
+      cy.location().then((currentLocation) => {
+        const urlPathName = currentLocation.pathname;
+        dealGroupID = urlPathName.split('/').pop(); // Grabbing Deal-Group ID from URL
+      });
+    });
+
+    it('Retrieve the deal-group', () => {
+      const requestOptions = getRequest();
+      requestOptions.url += `/${dealGroupID}?with=deals.dealBuyer.bidder,kpi,siteList.siteListProperty,dealGroupBuyer.bidder,dealGroupBuyer.seat,dealGroupAdvertiser&with_detail=true`;
+
+      cy.request(requestOptions)
+        .then((resp) => {
+          assert.equal(resp.status, 200, 'response status value ');
+          assert.equal(resp.body.id, dealGroupID, 'deal group ID value ');
+          saleForceID = resp.body.salesforce_id;
+          dealGroupName = resp.body.name;
+          closedLoopReport = resp.body.closed_loop_report;
+          target = resp.body.target_spend;
+          optKpi = resp.body.kpi.optimizable_kpi;
+          optMeasurement = resp.body.kpi.optimizable_kpi_measurement_partner;
+          outcomeKpi = resp.body.kpi.outcome_kpi;
+          kpiPartner = resp.body.kpi.outcome_kpi_measurement_partner;
+          bidderID = resp.body.dealGroupBuyer.bidder_id;
+        });
+    });
+
```

Comment:
There should a testcase that verifies all the info that was entered during deal creation.  Similar to the flow of Altice Automation